src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -107,6 +107,11 @@ configurationRegistry.registerConfiguration({
 			'default': 'right',
 			'description': nls.localize({ comment: ['This is the description for a setting. Values surrounded by single quotes are not to be translated.'], key: 'editorOpenPositioning' }, "Controls where editors open. Select 'left' or 'right' to open editors to the left or right of the current active one. Select 'first' or 'last' to open editors independently from the currently active one.")
 		},
+		'workbench.editor.reuseIfOpen': {
```

Comment:
The name and description reads a bit weird, maybe this should be called `revealIfOpen`? Because the editor is not reused if opened, it is being revealed. That also aligns with the `revealIfVisible` option.

Description idea: 

> Controls if an editor is revealed in any of the visible groups if opened. If disabled, an editor will prefer to open in the currently active editor group. If enabled, an already opened editor will be revealed instead of opened again in the currently active editor group. Note that there are some cases where this setting is ignored, e.g. when forcing an editor to open in a specific group or to the side of the currently active group.